dataikuapi/dss/admin.py

```diff
@@ -798,6 +798,41 @@ def set_definition(self, definition):
             "PUT", "/admin/globalAPIKeys/%s" % self.key,
             body = definition)
 
+class DSSPersonalApiKey(object):
```

Comment:
@AFrancani I just saw in your PR that you had to do a doc PR for the introduction of a new class.
I bet I have to do the same for this one?

Reply:
Yep 👍